src/dymaptic.GeoBlazor.Core/Scripts/dotNetBuilder.ts

```diff
@@ -656,4 +667,56 @@ export function buildDotNetTimeExtent(timeExtent: any): any | null {
         start: timeExtent.start.toISOString(),
         end: timeExtent.end.toISOString()
     } as any;
+}
+
+export function buildDotNetDimensionDefinition(dimensionDefinition: DimensionalDefinition): any | null {
```

Comment:
I don't think any of these methods are actually used. The use case for a dot net builder would be if the data was loaded from a server via JS, and then needed converting back to .NET. If there's no calling code, I would delete these (sorry).

Reply:
removed these and the imports at the top of the file.